src/pages/Signup/index.js

```diff
@@ -21,55 +39,154 @@ function Signup() {
     return <Redirect to="/dashboard" />;
   }
 
-  const signUpFormHandler = e => {
+  const signUpFormHandler = async e => {
     e.preventDefault();
+
+    if (values.password !== values.cpassword) {
+      setSignupResponse({
+        message: 'Passwords Should Match',
+        show: true,
+        type: 'form-alert-danger'
+      });
+
+      setTimeout(() => {
+        setSignupResponse({
+          show: false
+        });
+      }, 4000);
+      return;
+    }
+
+    const config = {
```

Comment:
Abstract this into an actions folder. to keep the function clean, also that would help make functionalities easy to plug in and unplug.